source/help/settings/account-settings.md

```diff
@@ -88,14 +88,16 @@ Configure how names are displayed in Mattermost: nickname, username or full name
 Select if the text in the center channel is fixed width and centered, or full width. 
 
 #### Language
-Select what language Mattermost displays in the user interface. Options include: 
+Select what language Mattermost displays in the user interface. Options currently include: 
 
 - English
 - Spanish
 - Portuguese
 - French
 - Japanese
 
+More languages are being added, please check back later if your preferred language is not on the list.
```

Comment:
Propose instead: 

"If you’re interested in contributing translations to Mattermost, please read our documentation on [localization](http://docs.mattermost.com/developer/localization.html)."